pkg/util/rbacs.go

```diff
@@ -0,0 +1,57 @@
+package util
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	coreapi "k8s.io/api/core/v1"
+	rbacapi "k8s.io/api/rbac/v1"
+	kerrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/util/retry"
+	ctrlruntimeclient "sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/openshift/ci-tools/pkg/results"
+)
+
+func CreateRBACs(ctx context.Context, sa *coreapi.ServiceAccount, role *rbacapi.Role, roleBinding *rbacapi.RoleBinding, client ctrlruntimeclient.Client, retryDuration time.Duration) error {
+	if err := client.Create(ctx, sa); err != nil && !kerrors.IsAlreadyExists(err) {
+		return results.ForReason("creating_service_account").WithError(err).Errorf("could not create service account '%s'", sa.Name)
+	}
+	if err := client.Create(ctx, role); err != nil && !kerrors.IsAlreadyExists(err) {
+		return results.ForReason("creating_roles").WithError(err).Errorf("could not create role '%s'", role.Name)
+	}
+	if err := client.Create(ctx, roleBinding); err != nil && !kerrors.IsAlreadyExists(err) {
+		return results.ForReason("binding_roles").WithError(err).Errorf("could not create role binding '%s'", roleBinding.Name)
+	}
+
+	if err := retry.OnError(
```

Comment:
This is a lot simpler with `k8s.io/apimachinery/pkg/util/wait.Poll`

Reply:
Thanks. Let me check that.